pyfeelpp-all/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
this is a big constraint, most OS doesn't are to this level.

Reply:
this is not used currently and will be changed

Reply:
this is used only by cmake preset which works only at cmake 3.21.
python codes will move there and the cmakelists,txt will be updated properly

Reply:
it was added by mistake, I can of course lower the version but it has not impact for now except for those using cmake --preset

Reply:
So OK, If you are sure that cmake doesn't failed for a basic use (without preset and thus also lower version)

Reply:
the main cmakelist does not `add_subdirectory` pyfeelpp-all for now (only via preset)